src/expr.rs

```diff
@@ -932,7 +932,7 @@ struct ControlFlow<'a> {
     span: Span,
 }
 
-fn to_control_flow<'a>(expr: &'a ast::Expr, expr_type: ExprType) -> Option<ControlFlow<'a>> {
```

Comment:
I don't think we should do this (or the similar changes elsewhere) - having the explicit lifetime inidicates that ControlFlow contains borrowed data.